theta_kappa.dot

```diff
@@ -545,8 +658,23 @@ digraph "Theta Kappa Zeta Family Tree" {
   /*tk1172 [label="TK1172|Michael Joseph Ruth"]*/
   /*tk1171 [label="TK1171|Russell Carl Olson"]*/
   tk1170 [label="TK1170|Bob Meyer"] tk1170 -> tk1248
+  /*tk1169 [label="TK1169|"]*/
+  /*tk1168 [label="TK1168|"]*/
+  /*tk1167 [label="TK1167|"]*/
   tk1166 [label="TK1166|Dan Klein"] tk1166 -> tk1227
+  /*tk1165 [label="TK1165|"]*/
+  /*tk1164 [label="TK1164|"]*/
+  /*tk1163 [label="TK1163|"]*/
+  /*tk1162 [label="TK1162|Andy Grinstead"]*/ /* LB Reece Scott */ /* BB Dave Cummings */
```

Comment:
Why is this commented out?

Reply:
We don't know some of those people's tk numbers or their bigs/littles.